### PR TITLE
DOC: Create MAINTAINERS.md

### PR DESCRIPTION
**Pull request by DominicOram**
As discussed in person. This lists the people that are maintainers on ophyd

### CODE REVIEW

**Comment by DominicOram** [comment]
We decided it in person as a large group of NSLS-II, DLS and LCLS people, using basically the methodology you describe. I appreciate though that some people on the list may feel it's been thrust upon them and I should have been more descriptive in the PR description that this is effectively that request. I'll add everyone on the list as a reviewer and we won't merge until we're all happy. @danielballan maybe we should do the same process for the other `MAINTAINER.md` files?

We can discuss further on here or mattermost if people have concerns but my understanding was that the role isn't really any additional work above what people are doing anyway and that we wanted to err on the side of lots of people from lots of facilities.

**Review by ZLLentz — APPROVED** [review]
I'm :+1: on being a maintainer here

**Comment by prjemian** [comment]
@DominicOram Thanks for your response.  I'm just asking so the process becomes documented (resolves any questions in the future).

**Review by prjemian — APPROVED** [review]
I agree to be a maintainer.

**Review by tangkong — APPROVED** [review]
I am also ok to be a maintainer 👍

**Review by klauer — APPROVED** [review]

File: MAINTAINERS.md
Comment (coretl) [inline MAINTAINERS.md:5]:
```suggestion
```
Please can you take me off now ophyd-async exists...

**Comment by prjemian** [comment]
This approved PR languishes.  Can it be merged now?

**Comment by danielballan** [comment]
I think that it was @klauer's intention to step back, and that we should remove him from this list. I hope it goes without saying that future input from Ken would always be more than welcome anywhere in the project.

Maybe @ZLLentz or @tangkong can confirm. With that change, I think we should merge.

File: MAINTAINERS.md
Comment (ZLLentz) [inline MAINTAINERS.md:7]:
```suggestion
```

**Pull request merged**
